core/standalone/build.gradle

```diff
@@ -140,6 +145,16 @@ processResources {
     }
 }
 
+// This normalization has been added in order to get a cache hit
+// when running tasks whose consider as inputs the resources
```

Comment:
This comment is not clear to me

Reply:
It should be done with 80959f9832cce44ad64de3bee2a391f00a907e9e

Reply:
What would you think of

>  Ignores the property `git.build.time` in `git.properties` files to allow `compileScala` task to be cached
 
Is it more clear?